tests/end_to_end/cucumber/steps/steps.py

```diff
@@ -82,3 +95,10 @@ def assert_dict_allclose(actual, expected, rtol=1e-06, atol=0):
 def check_solution(context):
     expected_solution = {row['variable']: float(row['value']) for row in context.table}
     assert_dict_allclose(context.outputs["solution"]["values"], expected_solution)
+
+
+def parse_output_folder_from_logs(logs: bytes) -> str:
+    for line in logs.splitlines():
+        if b'Output folder : ' in line:
+            return line.split(b'Output folder : ')[1].decode('ascii')
```

Comment:
i guess you meant :"Optimization results available "

Reply:
you're right, I copied what I had done for simulator, but it's not right. yours is better